go/libkb/saltpack.go

```diff
@@ -137,3 +137,10 @@ func BoxPublicKeyToKeybaseKID(k saltpack.BoxPublicKey) (ret keybase1.KID) {
 	p := k.ToKID()
 	return keybase1.KIDFromRawKey(p, KIDNaclDH)
 }
+
+func checkSaltpackBrand(b string) error {
+	if b != KeybaseSaltpackBrand {
+		return KeybaseSaltpackError{}
+	}
+	return nil
+}
```

Comment:
@patrickxb asked more or less the same question. I figured it's better to make it more restrictive at first and then loosen